portfolio/src/main/webapp/pages/contact.js

```diff
@@ -136,11 +146,33 @@ const Contact = {
                                                                     hour: 'numeric', minute: 'numeric', second: 'numeric' });
             this.comments.unshift(Object.assign({}, comment));
         },
-        // Removes the local comment that was most recently added, and displays an error message
-        removeLastCommentAndShowMessage(msg){
+        // Removes the local comment that was most recently added
```

Comment:
Nit. End the comment with a period (also applies for below)

Reply:
Fixed!